source/agent/quic/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "OWT-Server-QUIC-Agent",
+  "version": "4.3.0",
```

Comment:
update to 4.4.0

Reply:
It's 4.3 for now. I'll update it to the same value as other agents.